scripts/run_portserver.py

```diff
@@ -528,7 +540,7 @@ def main(args=None) -> None:
         )
         sys.exit(1)
 
-    request_handler = PortServerRequestHandler(ports_to_serve)
+    request_handler = PortServerRequestHandler(list(ports_to_serve))
```

Comment:
Why is the cast needed? Can _parse_port_ranges() provide the correct thing or is that a bad idea for some reason?

Reply:
> Can _parse_port_ranges() provide the correct thing

Fixed